src/profile_manager/tests/test_views.py

```diff
@@ -9,6 +9,10 @@
 from hackerspace_online.tests.utils import ViewTestUtilsMixin
 from siteconfig.models import SiteConfig
 
+from profile_manager.forms import ProfileForm
+
+from hackerspace_online.tests.utils import generate_form_data
+
 
 class ProfileViewTests(ViewTestUtilsMixin, TenantTestCase):
```

Comment:
Please add a test that a non-staff member that accesses this view will be given a ProfileForm and another that a staff member will be given the other form.

Reply:
im not sure how to directly test which form a user is given. But i already have something like that where a user tries to force a post request with variables from the staff-only form

Reply:
Check this out:
https://www.obeythetestinggoat.com/book/chapter_simple_form.html#_using_the_form_in_our_views

Basically you do a GET request, then check the context data contains the expected form.  This will be more interesting now, cus you'll need one test for staf that checks that BOTH forms are present.  And another test for students.

Reply:
nice! will do that